.CI/create_feedstocks.py

```diff
@@ -137,8 +137,8 @@ def write_token(name, token):
     subprocess.check_call(['git', 'status'])
     if removed_recipes:
         subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
-        msg = ('Removed recipe{s} ({}) after converting into feedstock{s}.'
-               ''.format(', '.join(removed_recipes),
+        msg = ('Removed recipe{s} ({}) after converting into feedstock{s}. '
+               '[skip appveyor]'.format(', '.join(removed_recipes),
```

Comment:
All of them should take `[ci skip]`. I think Travis will not take `[skip ci]` yet.